matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/handler/PresenceSyncHandler.kt

```diff
@@ -31,26 +31,24 @@ import javax.inject.Inject
 internal class PresenceSyncHandler @Inject constructor(private val matrixConfiguration: MatrixConfiguration) {
 
     fun handle(realm: Realm, presenceSyncResponse: PresenceSyncResponse?) {
-        if (matrixConfiguration.presenceSyncEnabled) {
```

Comment:
Here you removed the enable/disable logic. So now even with presence disabled the code will run.

Reply:
Yes this is on purpose. Since using a user setting, I wanted Realm to store Presence data regardless, so toggling is just a matter of refreshing the UI rather than having to force re-sync just to get Presence data we ignored before.

Reply:
If its the expected behaviour its fine, thanks